zen_queries/tests/tests.py

```diff
@@ -106,6 +107,17 @@ def test_template_response(self):
             response.render()
 
 
+class TemplateTagTestCase(TestCase):
+    def test_queries_disabled_template_tag(self):
+        widgets = Widget.objects.all()
+        with self.assertRaises(QueriesDisabledError):
+            django_render(None, "queries_disabled_tag.html", {"widgets": widgets})
+
+    def test_queries_dangerously_enabled_template_tag(self):
+        widgets = Widget.objects.all()
+        render(None, "queries_dangerously_enabled_tag.html", {"widgets": widgets})
```

Comment:
did you mean to use `django_render` for this line?

Reply:
No.. this one is using the `render` that _does_ disable queries, but then the tag re-enables them.